perfkitbenchmarker/benchmarks/mesh_network_benchmark.py

```diff
@@ -113,6 +114,8 @@ def RunNetperf(vm, benchmark_name, servers, result):
 
   match = re.findall(r'(\d+\.\d+)\s+\n', output)
   value = 0
+  if len(match) != (len(servers) - 1) * FLAGS.num_connections:
+    raise errors.Benchmarks.RunError('Netserver not started on remote vm.')
```

Comment:
Thanks for point out. It is not accurate. Netserver not reachable should be a better error message.